pkg/elemental/pages/index.vue

```diff
@@ -255,16 +258,18 @@ export default {
         v-if="isElementalOpNotInstalledAndHasSchema"
         class="mb-20"
         color="warning"
+        data-testid="warning-not-install-with-schema"
         v-html="t('product.notInstalledHasSchema', {}, true)"
       />
-      <h1 class="title">
+      <h1 class="title" data-testid="elemental-main-title">
         {{ t('elemental.menuLabels.titleDashboard') }}
       </h1>
       <!-- Top cards -->
       <div class="main-card-container">
         <div
           v-for="card, index in cards"
           :key="index"
+          :data-testid="`card-${card.title.toLowerCase().replace(/\s/g, '-')}`"
```

Comment:
can you create a `method` to generate this `data-testid` with fallbacks in case any of the 2 steps fails (`toLowerCase` and `replace`)? It's not only cleaner on the template part but it's also safer in terms preventing ui crashes 🙏